docs/rewards/rewards-claiming.mdx

```diff
@@ -78,3 +78,39 @@ To claim your rewards:
  5. If you require additional support to bridge your assets, you can refer to Arbitrum’s tutorial [here](https://docs.arbitrum.io/getting-started-users).
 
  **This Guide is applicable only on desktop with the MetaMask browser extension already installed.**
+
+
+<hr/>
+
+### Alternative Method to Claim Rewards
+
+<ol className="claim-steps">
+  <li>
+    Navigate to the <a href="https://claim.weatherxm.com/" target="_blank" rel="noopener noreferrer">Claim dApp</a>.
+    <img style={{ maxWidth: '650px', height: 'auto', margin: '10px 0' }} src={useBaseUrl("/img/project/web-claim-arbitrum.png")} alt="My Profile" />
+  </li>
+  
+  <li>
+    Connect your Wallet with the WalletConnect option.
+    <img style={{ maxWidth: '250px', height: 'auto', margin: '10px 0' }} src={useBaseUrl("/img/wallet/Wallet-connect.png")} alt="Wallet Connect" />
+  </li>
+
+  <li>
+    Scan the QR code with your mobile wallet.
```

Comment:
What will happen here? We need a sentence explaining what "Scan the QR code" means, e.g. that then you need to pick your mobile crypto wallet to connect it with WalletConnect etc.